### PR TITLE
add helmbuildbase build

### PR DESCRIPTION
**Pull request by tuannvm**
Alpine image with Helm-included. Use for repository create & upload.

### CODE REVIEW

File: helmbuildbase/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.4
```
Comment (MacTynow) [inline helmbuildbase/Dockerfile]: How about using a later version (like 3.6), which doesn't have vulnerabilities?

File: helmbuildbase/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+RUN apk --update add py-pip bash jq curl tar && \
```
Comment (MacTynow) [inline helmbuildbase/Dockerfile]: If this is just to rub builds, maybe bash and jq are not needed ?

Comment (so0k) [inline helmbuildbase/Dockerfile]: script uses bash - https://github.com/honestbee/helm-charts/pull/64/files#diff-4b4b60f5c2fffe5b6a2d75bb86782695

Comment (MacTynow) [inline helmbuildbase/Dockerfile]: Ok. You can dismiss this one :)

Comment (tuannvm) [inline helmbuildbase/Dockerfile]: Fix @MacTynow @so0k

**Pull request merged**